src/Flex.php

```diff
@@ -416,6 +436,7 @@ public static function getSubscribedEvents(): array
             ScriptEvents::POST_CREATE_PROJECT_CMD => 'configureProject',
             ScriptEvents::POST_INSTALL_CMD => 'install',
             ScriptEvents::POST_UPDATE_CMD => 'update',
+            PluginEvents::COMMAND => 'inspectCommand',
```

Comment:
@nicolas-grekas looks like v1.0.55 is broken as the `PluginEvents` class does not exist. Did you mean `CommandEvent`?